lib/common.py

```diff
@@ -100,9 +100,15 @@ def call(command, withSlashLogger = True):
     logger = lambda x: None
 
   def readproc(proc):
-    for line in iter(proc.stdout.readline, ''):
-      readproc.output += line
-      logger(line.rstrip('\n'))
+    import sys
+    for line in iter(proc.stdout.readline, b''):
+      # check python version
+      if sys.version_info.major == 2:
+        readproc.output += line
+        logger(line.rstrip('\n'))
+      else:
+        readproc.output += line.decode("utf-8")
+        logger(line.decode("utf-8").rstrip().rstrip('\n'))
```

Comment:
This block of code should work for both python 2 and 3, right?

Reply:
Hmmm... I guess this block does not always work on both versions.

Can you try to create Popen object with universal_newlines=True parameter (i.e. it will force Popen to use stdout/err in text mode on both versions of python)... and without the readproc changes?